src/styles/blocks/process.scss

```diff
@@ -0,0 +1,316 @@
+      transition: all $transition-duration;
+
+      @include theDesign {
+        width: 264px;
+        height: 264px;
+      }
+
+      &:hover {
+        transform: scale(1.1) translateY(-50%);
+        box-shadow: 0 0 20px $yellow-color;
+      }
+    }
+
+    &--green {
+      box-sizing: border-box;
+      height: 264px;
+      background-color: $white-color;
+      box-shadow: 0 9px 18px rgba(37, 41, 49, 0.03);
+      border-radius: 16px;
+      padding: 32px;
+      transition: all $transition-duration;
+
+      @include theDesign {
+        width: 264px;
+        height: 264px;
+      }
+
+      &:hover {
+        transform: scale(1.1) translateY(-50%);
+        box-shadow: 0 0 20px $green-color;
+      }
+    }
+
+    &--pink {
+      box-sizing: border-box;
+      height: 264px;
+      background-color: $white-color;
+      box-shadow: 0 9px 18px rgba(37, 41, 49, 0.03);
+      border-radius: 16px;
+      padding: 32px;
+      transition: all $transition-duration;
+
+      @include theDesign {
+        width: 264px;
+        height: 264px;
+      }
+
+      &:hover {
+        transform: scale(1.1) translateY(-50%);
+        box-shadow: 0 0 20px $pink-color;
+      }
+    }
+  }
+
+  &__num {
+    @extend %poppinsIconBlue;
+    width: 46px;
+    height: 24px;
+    border-radius: 23px;
+    background-color: $light-color;
+    text-align: center;
+    text-transform: uppercase;
+    color: $blue-color;
+
+    &--blue {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $light-blue-color;
+      background-color: $blue-gray-color;
+    }
+
+    &--yellow {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $yellow-color;
+      background-color: $dark-yellow-color;
+    }
+
+    &--green {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $green-color;
+      background-color: $dark-green-color;
+    }
+
+    &--pink {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $pink-color;
+      background-color: $dark-pink-color;
+    }
+  }
+
+  &__text {
+    @extend %openSansContent13;
+    color: $gray-color;
+    padding: 24px 0 32px;
+  }
+
+  &__learn-more {
+    @extend %poppinsButtonText;
+    line-height: 40px;
+    display: block;
+    background-image: url(../images/services/Arrow.svg);
+    background-size: 40px;
+    background-repeat: no-repeat;
+    text-decoration: none;
+    color: $gray-blue-color;
+
+    &::after {
+      content: "Learn more";
+      position: relative;
+      left: 56px;
+    }
+
+    &--blue {
```

Comment:
Same for button modifiers, remove classes or combine in one common class and add modifiers if needed